assets/laion.yaml

```diff
@@ -49,3 +103,54 @@
     from using LAION-5B for non-research purposes.
   monitoring: ''
   feedback: ''
+
+- type: dataset
+  name: LAION-2B-en
+  # General
+  organization: LAION
+  description: LAION-2B-en is a subset of the LAION-5B dataset and contains 2.3
+    billion English image-text pairs.
+  created_date:
+    value: 2022-12-12
+    explanation: The date the [[blog post]](https://laion.ai/blog/laion-5b/) was
+      released. Note that the dataset was already compiled earlier, e.g. the Stable
+      Diffusion model released earlier was trained on a subset of LAION-5B
+  url: https://arxiv.org/pdf/2210.08402.pdf
+  datasheet: https://laion.ai/blog/laion-5b/
+  modality: text (English) and image
+  size: 2.32B image-text pairs
+  sample: []
+  analysis: ''
+  # Construction
+  dependencies: [LAION-5B]
```

Comment:
should there be dependencies on clip and mclip due to filtering?

Reply:
You're right, but I think only CLIP for LAION-2B-en as it's english-only dataset, "cosine between text and image ViT-B/32 embeddings, clip for en, mclip for multi and nolang". However both CLIP and mCLIP should be added as dependencies of LAION-5B.